lib/utils/promote.ts

```diff
@@ -117,26 +109,29 @@ async function configure(deviceIp: string, config: any): Promise<void> {
 	const json = JSON.stringify(config);
 	const b64 = Buffer.from(json).toString('base64');
 	const str = `"$(base64 -d <<< ${b64})"`;
-	await execCommand(deviceIp, `os-config join '${str}'`, 'Configuring...');
+	await execCommand(deviceIp, `os-config join ${str}`, 'Configuring...');
```

Comment:
The single quotes around `'${str}'` are one of the reasons why `balena join` was failing to run on Windows. On macOS and Linux, `/bin/sh` was removing the single quotes when running 'ssh' as intended by the original developer. But on Windows, `cmd.exe` does not remove the single quotes.